src/styles/CHANGELOG.md

```diff
@@ -1,129 +1,177 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 This repo is in Prerelease. When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
 ========
+
+## [0.3.5]
```

Comment:
another useless piece of information for now until when Lerna is removed - the versioning between `react-components` and `styles` is not the same, this should be `0.0.22`. 
(No biggie!)